openfga/v1/openfga.proto

```diff
@@ -7,6 +7,30 @@ import "google/protobuf/timestamp.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 import "validate/validate.proto";
 
+// Object represents an OpenFGA Object.
+//
+// An Object is composed of a type and identifier (e.g. 'document:1')
+//
+// See https://openfga.dev/docs/concepts#what-is-an-object
+message Object {
+  string type = 1 [
+    (validate.rules).string = {
+      pattern: "^[^:#@\\s]{1,254}$"
+    },
+    (google.api.field_behavior) = REQUIRED,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      example: "\"document\""
+    }
+  ];
+
+  string id = 2 [
+    (validate.rules).string = {
+      pattern: "[^#:\\s]+$"
```

Comment:
Can someone verify this is correct? It seems that our only validation rules on an object id is that it can't contain the `#`, `:`, or `\s` characters. We don't have limits on the length of the id or format other than that though.

https://github.com/openfga/openfga/blob/main/pkg/tuple/tuple.go#L20

Is that correct?

Reply:
We do have length validation here: https://github.com/openfga/api/blob/72c90f6e861dc273750d5b13fbdc0e1259a9246a/openfga/v1/openfga.proto#L11

Reply:
Right, but what I wanted to verify is the regex above. It seems that is the second half of the objectRegex I referenced in the link earlier.

Could you confirm that?